src/syft/core/broadcast_message.py

```diff
@@ -0,0 +1,37 @@
+from syft.common.message import AbstractMessage
+from syft.common.id import UID
+from syft.core.io.route import Route
+
+class SyftBroadcastMessage(AbstractMessage):
+    def to_dict(self):
+        """
+        for testing, when protobuf is unavailable.
+        """
+        res = {}
+        for key, value in self.__dict__.items():
+            if hasattr(value, '__dict__'):
+                value = value.__dict__
+            res.update({key: value})
+        res['class'] = self.__class__.__name__
+        return res
+
+class BroadcastMessageSpecs(object):
```

Comment:
The specs should be more tightly integrated into teh particular message type you're using instead of being their own thing